client/src/components/ProTx/components/charts/MaltreatmentChart.js

```diff
@@ -16,7 +16,7 @@ function MaltreatmentChart({
 }) {
   if (selectedGeographicFeature && maltreatmentTypes.length !== 0) {
     return (
-      <div className="maltreatment-report">
+      <div className="maltreatment-chart">
```

Comment:
are these needed?  i can't seem to find maltreatment-chart or maltreatment report

Reply:
I kept them in there for sanity because they redundantly exist across all three plot components and should be refactored down to a single reusable set of selectors (instead of being plot-specific prefixed). To make the refactor easier (find & replace for the most part, before some structural cleanup) I wanted to leave them in place so they all mirrored one another.

The changes to observed-features was the same, alignment  before the structural refactoring.

I have a ticket for this task (COOKS-203) but wanted to wrap all the higher priority visible changes before tackling structural refactors under the hood.